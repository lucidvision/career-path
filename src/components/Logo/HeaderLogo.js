import React from 'react'
import { StyleSheet, Image, View } from 'react-native'

export default function HeaderLogo (props) {
  return (
    <View style={styles.container}>
      <Image
        style={styles.logo}
        resizeMode='contain'
        source={require('../../../assets/header.png')} />
    </View>
  )
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center'
  },
  logo: {
    height: 28,
    width: 260,
    alignSelf: 'center'
  }
})
