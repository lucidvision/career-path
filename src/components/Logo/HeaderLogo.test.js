import React from 'react'
import { shallow } from 'enzyme'
import HeaderLogo from './HeaderLogo'

describe('HeaderLogo', () => {
  const component = shallow(<HeaderLogo />)

  it('renders correctly', () => {
    expect(component).toMatchSnapshot()
  })
})
