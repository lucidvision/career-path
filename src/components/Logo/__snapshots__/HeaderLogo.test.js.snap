// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderLogo renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
    }
  }
>
  <Image
    resizeMode="contain"
    source={1}
    style={
      Object {
        "alignSelf": "center",
        "height": 28,
        "width": 260,
      }
    }
  />
</View>
`;
